src/sonic-eventd/rsyslog_plugin_tests/rsyslog_plugin_ut.cpp

```diff
@@ -0,0 +1,125 @@
+#include <iostream>
+#include <fstream>
+#include <regex>
+#include "gtest/gtest.h"
+#include "common/json.hpp"
+#include "common/events.h"
+#include "rsyslog_plugin/rsyslog_plugin.h"
+#include "rsyslog_plugin/syslog_parser.h"
+
+using namespace std;
+using json = nlohmann::json;
+
+RsyslogPlugin* plugin = NULL;
```

Comment:
Can you make this non global?
Create plugin can return.
You may use shared_ptr too, which will auto delete, when going out of scope.

Reply:
Done